data/techs.dat

```diff
@@ -239,3 +239,8 @@ danger = 1
 cost_list = 5000 | 45967 | 0
 pre_list = Heat Signature Reduction
 danger = 1
+
+[Knownledge Preservation]
+cost_list = 50000 | 50000 | 0
+pre = Advanced Autonomous Vehicles | Advanced Database Manipulation
```

Comment:
This needs to be a pre_list, per Travis.

Reply:
Yes, thx !